kong/plugins/oauth2/access.lua

```diff
@@ -365,7 +369,8 @@ local function authorize(conf)
             authenticated_userid = parameters[AUTHENTICATED_USERID],
             scope = scopes,
             challenge = challenge,
-            challenge_method = challenge_method
+            challenge_method = challenge_method,
+            plugin_identifier = get_plugin_identifier(conf)
```

Comment:
nit/non-blocker: should add a trailing comma here so that future diffs are a single line